garrysmod/gamemodes/terrortown/gamemode/lang/german.lua

```diff
@@ -194,7 +194,7 @@ L.search_dmg_tele   = "Es scheint, als sei ihre DNA durch Tachyonen verunstaltet
 L.search_dmg_car    = "Als diese Person die Straße überquerte, wurde sie von einem rücksichtslosen Fahrer überrollt."
 L.search_dmg_other  = "Du kannst keinen spezifischen Grund für den Tod dieser Person finden."
 
-L.search_weapon = "Es scheint, als wurde ein(e) {weapon} benutzt um sie zu töten."
+L.search_weapon = "Es scheint, als wurde {weapon} benutzt um sie zu töten."
```

Comment:
This sounds weird but is used in a majority of games and always using "ein(e)" does not maky any sense either.
Making it gender-neutral is a better way as using the old behaviour.